specs/zkevm-prover.md

```diff
@@ -130,24 +56,34 @@ interface ZKVerifier {
 }
 ```
 
-## Implementation
+## Prover as an RPC Server
 
-See [kroma-prover](https://github.com/kroma-network/kroma-prover) for details.
+[kroma-prover](https://github.com/kroma-network/kroma-prover) is implemented as a gRPC server that generates zkevm
+proofs for the requested height block. It can be utilized as a local component on the Challenger's machine
+or as a remote [gRPC] server (prover as a service).
 
-## RPC
+- Step 1:  The user (e.g., Challenger) requests a zkevm-proof for a block of specific height to kroma-prover with
+desired block height value.
+- Step 2: The kroma-prover obtains the trace of the corresponding block from kroma-geth RPC.  
+- Step 3: And then the kroma-prover generates zkevm-proof for target block, and return it to the User.
 
-Currently, to request a proof generation, a validator needs to communicate via [gRPC](https://grpc.io/). A validator acts
-as a client and prover acts as a server. Because proof generation takes time, it must wait for `FETCHING_TIMEOUT`
-seconds.
+Operating the grpc-kroma-prover, which is launched only when necessary, can alleviate the situation where
+regular validators and challengers need to allocate excessive system resources due to the proof generation process
+that is occasionally executed (perhaps rarely executed)
 
 ### Protobuf
 
-See [prover-grpc-proto](https://github.com/kroma-network/prover-grpc-proto) for details.
+The detailed gRPC specification can be available at [prover-grpc-proto].
 
 ## Summary of Definitions
 
 ### Constants
 
 | Name               | Value | Unit    |
-| ------------------ | ----- | ------- |
+|--------------------|-------|---------|
 | `FETCHING_TIMEOUT` | TBD   | seconds |
```

Comment:
We do not have this constant in this doc.

Reply:
The value are not fixed.

Reply:
Are you mean that table would be removed?

Reply:
I agree with you. Once the content is defined, let's recreate the table.